source/numerics.tex

```diff
@@ -8025,7 +8025,7 @@
 template, which is one-dimensional.
 The set of one-dimensional index values specified by a
 \tcode{gslice}
-are $$k = s + \sum_ji_jd_j$$
+are \[k = s + \sum_ji_jd_j\]
```

Comment:
Sure thing. Shall I also remove the comment on line 8029? Doesn't look too useful.